docs/concepts/registries.md

```diff
@@ -0,0 +1,82 @@
+Registries
+==========
+
+Registration is the process of taking the objects of a mod (items, blocks, sounds, etc.) and making them known to the game. Registering things is important, as without registration the game will simply not know about these objects in a mod and will exhibit great amounts of unexplainable (and probably crashy) behavior.
```

Comment:
"... as un-registered objects can behave in weird and seemingly inexplicable ways or might even make the game crash when used."